### PR TITLE
Add meanfi integrations - provides instructions for streaming functionality

### PR DESCRIPTION
**Pull request by welldan97**
Hi! I'm adding here meanfi integrations https://meanfi.com/
- Payment Stream: New account
- Payment Stream: Fund account
- Payment Stream: Withdraw funds
- Payment Stream: New stream
- Payment Stream: Transfer stream

### CODE REVIEW

File: utils/validations.tsx [hunk 1]
```diff
@@ -297,6 +297,264 @@ export const getCastleWithdrawSchema = () => {
...
+  })
+}
+export const getMeanWithdrawFromAccountSchema = ({
+  form,
+  connection,
+  mintInfo,
+}: {
+  form: any
+  connection: ConnectionContext
+  mintInfo: any
```
Comment (GregoryNEUT) [inline utils/validations.tsx:400]: You can type it as:

```
mintInfo?: MintInfo
```

needs to handle missing mintInfo a bit lower

**Review by GregoryNEUT — COMMENTED** [review]
Hello, I tried to make useful comments. Hope it helps!

**Comment by welldan97** [comment]
@GregoryNEUT , thanks for the feedback. I've updated the code

**Comment by welldan97** [comment]
Hi @GregoryNEUT, can we proceed with this PR? Do you want anything from my side?

**Comment by welldan97** [comment]
oh, I see thanks! 
@abrzezinski94 can we proceed with this?🙏

**Comment by abrzezinski94** [comment]
> oh, I see thanks! @abrzezinski94 can we proceed with this?🙏

Hi all good please fix conflicts and ill merge

**Comment by welldan97** [comment]
@abrzezinski94, It's ready

**Pull request merged**